homeassistant/components/media_player/samsungtv.py

```diff
@@ -22,7 +22,7 @@
 import homeassistant.helpers.config_validation as cv
 from homeassistant.util import dt as dt_util
 
-REQUIREMENTS = ['samsungctl[websocket]==0.7.1', 'wakeonlan==1.1.6']
+REQUIREMENTS = ['samsungctl[websocket]', 'wakeonlan==1.1.6']
```

Comment:
Latest version is 0.7.1, any idea of when that version will get bumped in samsungctl?

Reply:
https://github.com/Ape/samsungctl is not maintained. I have used a maintained fork https://github.com/kdschlosser/samsungctl which is tagged as version "0.8.0b". This is why I need this update

Reply:
In that case either we need to update pypi with newer versions, or make a fork of the whole project.
Have you tried to contact the author for the pypi crredentials?

Reply:
```
pip install samsungctl[websocket]
Collecting samsungctl[websocket]
  Using cached https://files.pythonhosted.org/packages/33/13/d97749371984ebacc53e66106cd3d224650aa2e33b21f3a4fcb4375435b3/samsungctl-0.7.1.tar.gz
Requirement already satisfied: websocket-client in /usr/local/lib/python2.7/site-packages (from samsungctl[websocket]) (0.53.0)
Requirement already satisfied: six in ./Library/Python/2.7/lib/python/site-packages (from websocket-client->samsungctl[websocket]) (1.11.0)
Building wheels for collected packages: samsungctl
  Running setup.py bdist_wheel for samsungctl ... done
  Stored in directory: /Users/name/Library/Caches/pip/wheels/3b/12/97/961e3e5cab297922729da21e130eb1b97af4dfa0f2a535a468
Successfully built samsungctl
Installing collected packages: samsungctl
Successfully installed samsungctl-0.7.1
```
So not pinned version will give you the same as pinned

Reply:
Yes, kdschlosser/samsungctl has to be installed manually `python setup.py install`. It is not that hard. I have opened an issue about original samsungctl repo here:
https://github.com/kdschlosser/samsungctl/issues/36

For now, there are only a few ways to fix it:
- approve this pull request and install the library manually
- rename kdschlosser library to samsungctl2
- Ape/samsungctl becomes active again and merges the code from kdschlosser. Though we were unable to contact him 

First method is the easiest and fastest one without any drawback, but it is only a temporary fix.

Reply:
asking users to manual install doesn't ... many users simply don't know how to do that, explaining is out of scope...